scripts/build_android_library.sh

```diff
@@ -36,28 +36,13 @@ build_android_native_library() {
 
   cmake . -DCMAKE_INSTALL_PREFIX="${CMAKE_OUT}" \
     -DCMAKE_TOOLCHAIN_FILE="${ANDROID_NDK}/build/cmake/android.toolchain.cmake" \
-    -DANDROID_ABI="${ANDROID_ABI}" \
+    --preset "android-${ANDROID_ABI}" \
     -DANDROID_PLATFORM=android-26 \
```

Comment:
Can you add this to preset as well?

Reply:
We need to build twice, one for x86_64, one for arm64-v8a.

Reply:
I think you can set one as default and override another one